linux/deploy_vm/wait_autoinstall_completed.yml

```diff
@@ -0,0 +1,16 @@
+# Copyright 2022 VMware, Inc.
+# SPDX-License-Identifier: BSD-2-Clause
+---
+- name: "Wait for autoinstall completed message"
```

Comment:
Our purpose is reducing and not use tasks delegate_to ESXi host, how about use URL like this "https://xx.xx.xx.xx/folder/test_file.log?dsName=datastore2&dcPath=ha-datacenter" to get the file content?

Reply:
`common/vm_wait_logbundle_msg.yml` was added to check or wait for a message in VM's log bundles.